src/Actions/CreateNewUser.php

```diff
@@ -52,15 +53,15 @@ public function create(array $input)
     private function buildValidator(array $input): ValidationValidator
     {
         $rules = [
-            'name'              => ['required', 'string', 'max:255', resolve(PoliteUsername::class)],
+            'name'              => ['required', 'string', 'min:3', 'max:30', resolve(PoliteUsername::class)],
```

Comment:
move min and max values to some constant so it can be reused in this file + in the test and easily updated if needed